integration/test/util/api.py

```diff
@@ -20,6 +20,16 @@ def reset(self) -> None:
         resp = requests.post(f"https://{self.host}/conf/reset")
         _handle_err(resp, 200, "Failed to reset the mock API")
 
+    def set_about_failure(self, status_code) -> None:
```

Comment:
These set_about functions don't seem to be used anywhere?

They both do pretty much the same thing, so if needed, you could just have one function `set_about(self, status_code)`.

Also, for consistency, the blank line between requests.post and _handle_err should be dropped.

Reply:
They are added in to the integration tests in order to check that the errors are being handled correctly by causing a bad api call

Reply:
I didn't look at the whole file and missed the pattern being used for success and failure functions, so for consistency there should be the original 2 functions you wrote for set_about. Sorry!!